src/Components/ContextualMenu/ContextualMenuItem.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components"
+
+import { themeGet } from "@styled-system/theme-get"
+import { Clickable, Separator } from "@artsy/palette"
+
+export interface ContextualMenuItemProps {
+  children: React.ReactNode
+  onClick?: () => void
+}
+
+export const ContextualMenuItem: React.FC<ContextualMenuItemProps> = ({
+  children,
+  onClick,
+}) => {
+  return (
+    <ContextualMenuItemContent onClick={onClick}>
+      {children}
+    </ContextualMenuItemContent>
+  )
+}
+
+const ContextualMenuItemContent = styled(Clickable).attrs({
+  display: "block",
+  width: "100%",
+  padding: 2,
+})`
+  user-select: none;
+
+  transition: background-color 150ms;
+  &:hover {
+    background-color: ${themeGet("colors.black5")};
+  }
```

Comment:
🧐 🧐 🧐 not sure where to find a standard for this.

<img width="400" alt="transitions" src="https://user-images.githubusercontent.com/140521/219855042-e68121ff-8d6b-4014-afe6-7d81b1f46027.png">

Reply:
The styling of the contextual menu  could in general benefit from some discussion — happy to follow up with Ryan and iterate in another PR.

Reply:
Good call. Generally speaking it should be 250. Can we move this to a `constants.ts` file in palette and export? Might be worth updating all of these in palette, too: 

<img width="742" alt="Screenshot 2023-02-22 at 2 51 19 PM" src="https://user-images.githubusercontent.com/236943/220778595-9b94b42f-d1a6-4bd0-b2cf-99131d55f5d6.png">